develop/devguide/GQI_Extensions/API_Reference/IGQIUpdater.md

```diff
@@ -21,6 +21,9 @@ It allows adding, removing, and updating rows, or updating individual cells.
 
 Adds a new row to the query result. If a row with the same row key already exists, that row will be updated instead. This method behaves as an *add-or-update* operation.
 
+> [!CAUTION]
+> Prior to DataMiner 10.5.0 [CU4]/10.5.7, if a row with the same row key already exists, calling the AddRow method will result in a duplicate row with the same row key being added.
```

Comment:
Is this change also included in 10.4.0 [CU16]? And do you know which release note is responsible for this change?

Reply:
LGTM